File: dotnet/cafdemalihapa/Directories/Hosting/Feature/WebLinks/WebLinksDirectory.cs
namespace cafdemalihapa.Directories.Hosting.Feature.WebLinks
{
    public class WebLinksDirectory()
    {
        public string GetPath()
        {
            var directoryName = "web links";
            var featureDirectoryPath = FeatureDirectory.GetPath();
            var notesAndMessages = Path.Combine(featureDirectoryPath, directoryName);
            return notesAndMessages;
        }

        public void Create()
        {
            var path = GetPath();
            if(!Directory.Exists(path))
            {
                Directory.CreateDirectory(path);
            }
        }
    }
}

File: dotnet/cafdemvers/zzq-cafdemwimu/cafdemwimu.console/Directories/Hosting/Feature/WebLinks/WebLinksDirectory.fs
namespace cafdemwimu.console.Directories.Hosting.Feature.WebLinks

open System.IO
open cafdemwimu.console.Directories.Hosting.Feature

type WebLinksDirectory() =
    member _.GetPath() =
        let directoryName = "web links"
        let featureDirectoryPath = FeatureDirectory.GetPath()
        let notesAndMessages = Path.Combine(featureDirectoryPath, directoryName)
        notesAndMessages

    member this.Create() =
        let path = this.GetPath()
        if not (Directory.Exists(path)) then
            Directory.CreateDirectory(path) |> ignore
